### PR TITLE
Improve SyntaxError message when an ERRORTOKEN is returned by the tokenizer

### PR DESCRIPTION
**Pull request by lysnikolaou**

### CODE REVIEW

**Comment by lysnikolaou** [comment]
One test that was failing before now succeeds. Seven more to go!

File: Parser/pegen/pegen.c [hunk 1]
```diff
@@ -38,6 +38,43 @@ byte_offset_to_character_offset(PyObject *line, int col_offset)
     return size;
 }
 
+int
```
Comment (pablogsal) [inline Parser/pegen/pegen.c:41]:
```suggestion
static int
```

File: Parser/pegen/pegen.c [hunk 2]
```diff
@@ -38,6 +38,43 @@ byte_offset_to_character_offset(PyObject *line, int col_offset)
...
+            msg = "invalid character in identifier";
+            break;
+        default:
+            msg = "unknown parsing error";
+    }
+
+
+    PyErr_Format(PyExc_SyntaxError, msg);
+    // There is no reliable column information for this error
+    PyErr_SyntaxLocationObject(p->tok->filename, p->tok->lineno, 0);
```
Comment (gvanrossum) [inline Parser/pegen/pegen.c:73]: Is that really true? The old parser does seem to point to the end of the line at least for E_EOFS and E_EOLS (I didn't check all the others).

Comment (lysnikolaou) [reply Parser/pegen/pegen.c:73]: That comment was already there. I'm going to look into this in more detail, as soon as we have fixed #47, which is currently in the way.

**Pull request merged**